JsonSchema.Data/Vocabularies.cs

```diff
@@ -31,27 +30,29 @@ public static void Register(VocabularyRegistry? vocabRegistry = null, SchemaRegi
 		schemaRegistry ??= SchemaRegistry.Global;
 
 		vocabRegistry.Register(Data);
-		SchemaKeywordRegistry.Register<DataKeyword>(DataExtSerializerContext.Default);
+		SchemaKeywordRegistry.Register<DataKeyword>(JsonSchemaDataSerializerContext.Default);
 		schemaRegistry.Register(MetaSchemas.Data);
 		schemaRegistry.Register(MetaSchemas.Data_202012);
 	}
 }
 
+/// <summary>
+/// A serializer context for this library.
+/// </summary>
 [JsonSerializable(typeof(DataKeyword))]
 [JsonSerializable(typeof(OptionalDataKeyword))]
 [JsonSerializable(typeof(Dictionary<string, string>))]
 [JsonSerializable(typeof(Dictionary<string, JsonNode>))]
-[JsonSerializable(typeof(JsonPointer))]
-[JsonSerializable(typeof(RelativeJsonPointer))]
 [JsonSerializable(typeof(Uri))]
-internal partial class DataExtSerializerContext : JsonSerializerContext
+public partial class JsonSchemaDataSerializerContext : JsonSerializerContext
 {
-	public static TypeResolverOptionsManager OptionsManager { get; }
+	internal static JsonSerializerOptions CombinedOptions { get; }
 
-	static DataExtSerializerContext() {
-		OptionsManager = new(
-			Default,
-			JsonSchema.TypeInfoResolver
-		);
+	static JsonSchemaDataSerializerContext()
+	{
+		CombinedOptions = new JsonSerializerOptions
```

Comment:
Why do we need to combine options anymore? This new strategy should avoid the need for it.

Reply:
This was the workaround for the bug.  I need it in the `UriIdentifier` class where I'm serializing a `JsonSchema`.

Reply:
Which bug? Can you include a comment / pointer to the issue if it's a bug in STJ?